modules/apps/portal/portal-default-permissions-web/src/main/java/com/liferay/portal/defaultpermissions/web/internal/configuration/display/PortalDefaultPermissionsGroupConfigurationScreenWrapper.java

```diff
@@ -0,0 +1,110 @@
+import com.liferay.portal.defaultpermissions.resource.PortalDefaultPermissionsModelResourceRegistry;
+import com.liferay.portal.defaultpermissions.web.internal.display.context.PortalDefaultPermissionsConfigurationDisplayContext;
+import com.liferay.portal.defaultpermissions.web.internal.display.context.PortalDefaultPermissionsGroupConfigurationDisplayContext;
+import com.liferay.portal.kernel.feature.flag.FeatureFlagManagerUtil;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.kernel.model.Group;
+import com.liferay.site.settings.configuration.admin.display.SiteSettingsConfigurationScreenContributor;
+import com.liferay.site.settings.configuration.admin.display.SiteSettingsConfigurationScreenFactory;
+
+import java.util.Locale;
+
+import javax.servlet.ServletContext;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Stefano Motta
+ */
+@Component(service = ConfigurationScreen.class)
+public class PortalDefaultPermissionsGroupConfigurationScreenWrapper
+	extends ConfigurationScreenWrapper {
+
+	@Override
+	protected ConfigurationScreen getConfigurationScreen() {
+		return _siteSettingsConfigurationScreenFactory.create(
+			new DefaultPermissionsSiteSettingsConfigurationScreenContributor());
+	}
+
+	@Reference
+	private Language _language;
+
+	@Reference
+	private PortalDefaultPermissionsModelResourceRegistry
+		_portalDefaultPermissionsModelResourceRegistry;
+
+	@Reference(
+		target = "(osgi.web.symbolicname=com.liferay.portal.defaultpermissions.web)"
+	)
+	private ServletContext _servletContext;
+
+	@Reference
+	private SiteSettingsConfigurationScreenFactory
+		_siteSettingsConfigurationScreenFactory;
+
+	private class DefaultPermissionsSiteSettingsConfigurationScreenContributor
+		implements SiteSettingsConfigurationScreenContributor {
+
+		@Override
+		public String getCategoryKey() {
+			return "default-permissions";
+		}
+
+		@Override
+		public String getJspPath() {
+			return "/configuration" +
+				"/portal_default_permissions_configuration.jsp";
+		}
+
+		@Override
+		public String getKey() {
+			return "portal-default-permissions-group-configuration";
+		}
+
+		@Override
+		public String getName(Locale locale) {
+			return _language.get(locale, "default-permissions");
+		}
+
+		@Override
+		public ServletContext getServletContext() {
+			return _servletContext;
+		}
+
+		@Override
+		public boolean isVisible(Group group) {
+			if (FeatureFlagManagerUtil.isEnabled("LPD-21265")) {
+				return true;
+			}
+
+			return false;
+		}
+
+		@Override
+		public void setAttributes(
+			HttpServletRequest httpServletRequest,
+			HttpServletResponse httpServletResponse) {
+
+			httpServletRequest.setAttribute(
+				PortalDefaultPermissionsConfigurationDisplayContext.class.
```

Comment:
should't we use WebKeys?

Reply:
I don't know, probably you're right. I found other example like this but let's see what Brian says on the PR already sent (the company one) where I'm using the same logic

Reply:
we have this WebKeys.PORTLET_DISPLAY_CONTEXT but it could be wrong, I agree, let's see what Brian says in his review